common/common.cc

```diff
@@ -486,7 +486,41 @@ vector<int> sorbet::findLineBreaks(string_view s) {
 class SetTerminateHandler {
 public:
     static void on_terminate() {
+        auto eptr = current_exception();
+        if (eptr) {
+            try {
+                // Aparently this is the only way to convert a std::exception_ptr to std::exception
+                std::rethrow_exception(eptr);
+            } catch (const std::exception &e) {
+                int status;
+                // nullptr to request that __cxa_demangle allocate the string on our behalf,
+                // instead of populating a pre-allocated buffer.
+                auto demangled = abi::__cxa_demangle(typeid(e).name(), nullptr, nullptr, &status);
```

Comment:
`common.cc` already has a `demangle` function. 😅

Reply:
I should have noticed that when I didn't need to add the `<cxxabi.h>` include!